aldryn_newsblog/tests/test_reversion.py

```diff
@@ -173,3 +175,31 @@ def test_edit_plugin_directly(self):
         response = self.client.get(article.get_absolute_url())
         self.assertContains(response, content1)
         self.assertNotContains(response, content2)
+
+    def test_blog_config_recovery_accessible(self):
+        with transaction.atomic():
```

Comment:
Couldn't find any reasons why it shouldn't. Seems that there is only one limitation with ``MySQL`` with not ``InnoDB``, but then transaction stuff is a noop, so shouldn't produce any issues.
Anyway this seems as a standard way of creating revision which ``django-reversion`` seems to use.